CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ librdkafka v2.2.1 is a maintenance release:
    are partition leader changes and a stale leader epoch is received (#4429).
  * Fix a segmentation fault when closing a consumer using the
    cooperative-sticky assignor before the first assignment (#4381).
+ * Fix for insufficient buffer allocation when allocating rack information (@wolfchimneyrock, #4449).
+
+
+## Fixes
+
+### General fixes
+
+ * An assertion failed with insufficient buffer size when allocating
+   rack information on 32bit architectures.
+   Solved by aligning all allocations to the maximum allowed word size.
```

Comment:
```suggestion
   Solved by aligning all allocations to the maximum allowed word size (#4449).
```

Reply:
That is usually put only in the summary list. We can change the policy, so it's easier to match the PRs to the bug descriptions